recipe/meta.yaml

```diff
@@ -20,7 +20,7 @@ source:
     - lz4-fix.patch
```

Comment:
Leaving this patch for now (#315). I think it requires the upstream PR https://github.com/TileDB-Inc/TileDB/pull/5021, which wasn't included in [2.25.0](https://github.com/TileDB-Inc/TileDB/releases/tag/2.25.0)